packages/legacy/app/ios/Podfile.lock

```diff
@@ -794,14 +795,15 @@ SPEC CHECKSUMS:
   RNGestureHandler: a479ebd5ed4221a810967000735517df0d2db211
   RNKeychain: ff836453cba46938e0e9e4c22e43d43fa2c90333
   RNLocalize: d4b8af4e442d4bcca54e68fc687a2129b4d71a81
-  RNPermissions: 7f93fea75d6ea684fe600b08e2c244098a49244b
+  RNPermissions: 868c4e459921873df0c756f37450457a88bf037c
   RNReanimated: ab2e96c6d5591c3dfbb38a464f54c8d17fb34a87
   RNScreens: 85d3880b52d34db7b8eeebe2f1a0e807c05e69fa
   RNSVG: d7d7bc8229af3842c9cfc3a723c815a52cdd1105
   RNVectorIcons: 8b5bb0fa61d54cd2020af4f24a51841ce365c7e9
   SocketRocket: f32cd54efbe0f095c4d7594881e52619cfe80b17
+  VisionCamera: b236f548d6507923857dade542c7e0a148ed3492
   Yoga: 86fed2e4d425ee4c6eab3813ba1791101ee153c6
 
 PODFILE CHECKSUM: 702646d9563d22bf09e336926a5fb0e2d3239409
 
-COCOAPODS: 1.13.0
+COCOAPODS: 1.11.2
```

Comment:
Able to keep the higher version of Cocoapods currently being used?

Reply:
yes